google-cloud-firestore/lib/google/cloud/firestore/concurrent/promises.rb

```diff
@@ -0,0 +1,93 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Google
+  module Cloud
+    module Firestore
+      ##
+      # @private
+      module Promise
```

Comment:
Why is this private ? shouldn't this be exposed to users as results ?

Reply:
Correct. Removed the private label from the class and added a brief description.